WcaOnRails/app/views/users/index.html.erb

```diff
@@ -3,21 +3,18 @@
 <div class="container">
   <h1><%= yield(:title) %></h1>
 
-  <%= grid(@users_grid) do |g|
+  <%= wca_table data: { toggle: "table", pagination: "true", side_pagination: "server", url: users_url,
+                        search: "true", sort_name: "name", undefined_text: "" } do %>
+    <thead>
+      <tr>
+        <th class="wca-id" data-field="wca_id" data-sortable="true" data-width="15%">WCA ID</th>
+        <th class="name" data-field="name" data-sortable="true" data-width="40%">Name</th>
+        <th class="email" data-field="email" data-sortable="true" data-width="40%">Email</th>
```

Comment:
It would apply to all bootstrap-tables, so they wouldn't look like 'wca tables'. You can see how it looks by importing 'bootstrap-table' in the scss file.